### PR TITLE
move oss images to partial

### PR DESCRIPTION
**Pull request by tahabi**
fairly self-explanatory

### CODE REVIEW

File: ocfweb/templates/partials/open-source-pics.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<p class="ocf-logos">
+  <a href="https://www.linuxfoundation.org/"><img src="{% static 'img/about/logos/linux.png' %}" alt="linux" title="Linux" /></a>
+  <a href="https://www.debian.org/"><img src="{% static 'img/about/logos/debian.png' %}" alt="debian" title="Debian" /></a>
+  <a href="https://puppet.com/"><img src="{% static 'img/about/logos/puppet.png' %}" alt="puppet" title="Puppet" /></a>
+  <a href="https://www.python.org/"><img src="{% static 'img/about/logos/python.png' %}" alt="python" title="Python" /></a>
+  <a href="https://www.apache.org/"><img src="{% static 'img/about/logos/apache.png' %}" alt="apache" title="Apache" /></a>
+  <a href="https://www.mariadb.org/"><img src="{% static 'img/about/logos/mariadb.png' %}" alt="mariadb" title="MariaDB" /></a>
+  <a href="https://www.xfce.org/"><img src="{% static 'img/about/logos/xfce.png' %}" alt="xfce" title="Xfce" /></a>
+  <a href="http://www.vim.org/"><img src="{% static 'img/about/logos/vim.png' %}" alt="vim" title="vim" /></a>
+  <a href="https://www.gnu.org/software/emacs/"><img src="{% static 'img/about/logos/emacs.png' %}" alt="emacs" title="Emacs" /></a>
```
Comment (chriskuehl) [inline ocfweb/templates/partials/open-source-pics.html]: 4 space indent please

**Comment by tahabi** [comment]
DRY is a holy commandment in the Church of Emacs

File: ocfweb/about/templates/about/staff.html [hunk 2]
```diff
@@ -94,15 +94,7 @@
...
-                    <a href="https://www.debian.org/"><img src="{% static 'img/about/logos/debian.png' %}" alt="debian" title="Debian" /></a>
-                    <a href="https://puppet.com/"><img src="{% static 'img/about/logos/puppet.png' %}" alt="puppet" title="Puppet" /></a>
-                    <a href="https://www.python.org/"><img src="{% static 'img/about/logos/python.png' %}" alt="python" title="Python" /></a>
-                    <a href="https://www.apache.org/"><img src="{% static 'img/about/logos/apache.png' %}" alt="apache" title="Apache" /></a>
-                    <a href="https://www.mariadb.org/"><img src="{% static 'img/about/logos/mariadb.png' %}" alt="mariadb" title="MariaDB" /></a>
-                    <a href="https://www.xfce.org/"><img src="{% static 'img/about/logos/xfce.png' %}" alt="xfce" title="Xfce" /></a>
-                    <a href="http://www.vim.org/"><img src="{% static 'img/about/logos/vim.png' %}" alt="vim" title="vim" /></a>
+                {% include 'partials/open-source-pics.html' %}
                 </p>
```
Comment (jvperrin) [inline ocfweb/about/templates/about/staff.html]: I think you missed removing a closing `</p>` tag here

File: ocfweb/about/templates/about/lab-open-source.html [hunk 3]
```diff
@@ -7,16 +7,7 @@
...
-                <img src="{% static 'img/about/logos/linux.png' %}" alt="linux" />
-                <img src="{% static 'img/about/logos/debian.png' %}" alt="debian" />
-                <img src="{% static 'img/about/logos/puppet.png' %}" alt="puppet" />
-                <img src="{% static 'img/about/logos/python.png' %}" alt="python" />
-                <img src="{% static 'img/about/logos/apache.png' %}" alt="apache" />
-                <img src="{% static 'img/about/logos/mariadb.png' %}" alt="mariadb" />
-                <img src="{% static 'img/about/logos/xfce.png' %}" alt="xfce" />
-                <img src="{% static 'img/about/logos/vim.png' %}" alt="vim" />
-            </p>
+        {% include 'partials/open-source-pics.html' %}
```
Comment (jvperrin) [inline ocfweb/about/templates/about/lab-open-source.html]: This should be indented so it lines up with the other elements in the `<div>`.

File: ocfweb/templates/partials/open-source-pics.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+<p class="ocf-logos">
+    <a href="https://www.linuxfoundation.org/"><img src="{% static 'img/about/logos/linux.png' %}" alt="linux" title="Linux" /></a>
+    <a href="https://www.debian.org/"><img src="{% static 'img/about/logos/debian.png' %}" alt="debian" title="Debian" /></a>
+    <a href="https://puppet.com/"><img src="{% static 'img/about/logos/puppet.png' %}" alt="puppet" title="Puppet" /></a>
+    <a href="https://www.python.org/"><img src="{% static 'img/about/logos/python.png' %}" alt="python" title="Python" /></a>
+    <a href="https://www.apache.org/"><img src="{% static 'img/about/logos/apache.png' %}" alt="apache" title="Apache" /></a>
+    <a href="https://www.mariadb.org/"><img src="{% static 'img/about/logos/mariadb.png' %}" alt="mariadb" title="MariaDB" /></a>
+    <a href="https://www.xfce.org/"><img src="{% static 'img/about/logos/xfce.png' %}" alt="xfce" title="Xfce" /></a>
+    <a href="http://www.vim.org/"><img src="{% static 'img/about/logos/vim.png' %}" alt="vim" title="vim" /></a>
+    <a href="https://www.gnu.org/software/emacs/"><img src="{% static 'img/about/logos/emacs.png' %}" alt="emacs" title="Emacs" /></a>
```
Comment (jvperrin) [inline ocfweb/templates/partials/open-source-pics.html]: Hmmmmmmmmm

Comment (tahabi) [inline ocfweb/templates/partials/open-source-pics.html]: sssshhh, before someone notices

**Pull request merged**